test/support/sample.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "left": 0.11960784313725491,
+    "top": 0.17484848484848484,
+    "width": 0.14705882352941174,
+    "height": 0.009696969696969704,
+    "chars": "operations@example.com"
+  },
+  {
+    "left": 0.9941176470588236,
+    "top": 0.2387878787878788,
+    "width": 0.003529411764705892,
+    "height": 0.010303030303030286,
+    "chars": "{"
+  },
+  {
+    "left": 0.6874509803921569,
+    "top": 0.24818181818181817,
+    "width": 0.09529411764705875,
+    "height": 0.007575757575757569,
+    "chars": "INVOICE# 2599"
+  },
+  {
+    "left": 0.7156862745098039,
+    "top": 0.26484848484848483,
+    "width": 0.10156862745098039,
+    "height": 0.007878787878787874,
+    "chars": "DATE 04/02/2017"
+  },
+  {
+    "left": 0.6874509803921569,
```

Comment:
Since we don't know the dimensions of the font's characters, there is no way to accurately calculate the width/top/left dimensions - so I left that part out and just copied the same dimensions